### PR TITLE
initial commit to create a hardcoded intent handler for nlp

### PR DESCRIPTION
**Pull request by sizov**

### CODE REVIEW

File: src/client/src/apps/SimpleLauncher/applicationConfigurations.tsx [hunk 1]
```diff
@@ -121,6 +121,19 @@ export const appConfigs: ApplicationConfig[] = [
       },
     },
   },
+  {
+    name: 'Spotlight',
+    url: `${ORIGIN}/spotlight`,
+    icon: limitCheckerIcon,
```
Comment (fdore) [inline src/client/src/apps/SimpleLauncher/applicationConfigurations.tsx]: there is a `searchIcon` you can use

File: src/client/src/apps/SimpleLauncher/applicationConfigurations.tsx [hunk 2]
```diff
@@ -121,6 +121,19 @@ export const appConfigs: ApplicationConfig[] = [
       },
     },
   },
+  {
```
Comment (fdore) [inline src/client/src/apps/SimpleLauncher/applicationConfigurations.tsx]: I would restrict it to `process.env.NODE_ENV === 'development'` for now

File: src/client/src/apps/SpotlightRoute/Spotlight.tsx [hunk 3]
```diff
@@ -7,6 +7,8 @@ import { AutobahnConnectionProxy } from 'rt-system'
 import { take, timeout } from 'rxjs/operators'
 import { mapIntent } from './responseMapper'
 import { DetectIntentResponse } from 'dialogflow'
+import { usePlatform } from '../../rt-platforms'
```
Comment (fdore) [inline src/client/src/apps/SpotlightRoute/Spotlight.tsx]: => `import { usePlatform } from 'rt-platforms'`

File: src/client/src/apps/SpotlightRoute/handleIntent.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
+import { DetectIntentResponse } from 'dialogflow'
+import {
+  CURRENCY_INTENT,
+  MARKET_INFO_INTENT,
+  SPOT_QUOTE_INTENT,
+  TRADES_INFO_INTENT,
+} from './intents'
+import { BasePlatformAdapter } from '../../rt-platforms/platformAdapter'
+import { WindowConfig } from '../../rt-platforms'
+import { WindowCenterStatus } from '../../rt-components/tear-off/types'
```
Comment (fdore) [inline src/client/src/apps/SpotlightRoute/handleIntent.ts]: same here

File: src/client/src/apps/SimpleLauncher/applicationConfigurations.tsx (hunk 1 above)
Comment (sizov) [reply src/client/src/apps/SimpleLauncher/applicationConfigurations.tsx]: done

File: src/client/src/apps/SpotlightRoute/Spotlight.tsx (hunk 3 above)
Comment (sizov) [reply src/client/src/apps/SpotlightRoute/Spotlight.tsx]: done

File: src/client/src/apps/SpotlightRoute/handleIntent.ts (hunk 4 above)
Comment (sizov) [reply src/client/src/apps/SpotlightRoute/handleIntent.ts]: done

File: src/client/src/apps/SimpleLauncher/applicationConfigurations.tsx (hunk 2 above)
Comment (sizov) [reply src/client/src/apps/SimpleLauncher/applicationConfigurations.tsx]: done

**Pull request merged**